lioncore/metametamodel/lioncore.json

```diff
@@ -0,0 +1,2196 @@
+{
+  "serializationFormatVersion": "1",
+  "languages": [
+    {
+      "key": "LIonCore-M3",
+      "version": "1"
+    }
+  ],
+  "nodes": [
+    {
+      "id": "-id-LIonCore-M3",
+      "concept": {
+        "language": "LIonCore-M3",
+        "version": "1",
+        "key": "Language"
+      },
+      "properties": [
+        {
+          "property": {
+            "language": "LIonCore-M3",
+            "version": "1",
+            "key": "LIonCore-builtins-INamed-name"
+          },
+          "value": "LIonCore.M3"
```

Comment:
And here we use a dot. I am confused as to where (and why) we use dot, minus and underscore.

Reply:
This is the name of the language. It has no technical meaning, so could also be "LIonWeb LIonCore M3 Metamodel" or anything else.
As naming is hard, I chose this one without any imagination. And made it different from _key_ and _id_ for aforementioned reason.

Reply:
I agree naming is hard indeed !